options.go

```diff
@@ -44,11 +44,20 @@ func WithSkipPopulateStruct(skipPopulateStruct bool) UnmarshalOption {
 	}
 }
 
+// WithExcludeKnownTypeFromMap is an UnmarshalOption function to set the excludeKnownTypesFromMap option.
+// Exclude known type is set to false by default.
+func WithExcludeKnownTypeFromMap(excludeKnownTypes bool) UnmarshalOption {
+	return func(options *unmarshalOptions) {
+		options.excludeKnownTypesFromMap = excludeKnownTypes
+	}
+}
+
 type UnmarshalOption func(*unmarshalOptions)
 
 type unmarshalOptions struct {
-	mode               Mode
-	skipPopulateStruct bool
+	mode                     Mode
+	skipPopulateStruct       bool
+	excludeKnownTypesFromMap bool
```

Comment:
I'd call it `excludeKnownFieldsFromMap`
WDYT?

Reply:
and then also align [this func](https://github.com/PerimeterX/marshmallow/pull/20/files#diff-4eec45b6a922746411c0f8aea183e30f77af1b0a5bea1688eaf343a5f9c23703R49)

Reply:
👍  changed in all places `excludeKnownTypes` to `excludeKnownFields`